src/engine/dom-engine.js

```diff
@@ -288,27 +285,21 @@ $DE.injectElementStyles = function(elm, general_class, instance_class) {
     var props = $DE.getAnmProps(elm);
     props.gen_class  = general_class;
     props.inst_class = instance_class;
```

Comment:
Seems this data is never used now, so `injectElementStyles` is not needed at all?